modules/mauth-proxy/README.md

```diff
@@ -1,35 +1,45 @@
-## Goal:
-
-The goal of this simple service is to append valid mAuth headers to any incoming request and send this (now authenticated) request to the destination service.
-
-## General use:
-Using [Typesafe Config](https://github.com/typesafehub/config)
-Create `application.conf` file on your classpath with following settings
-```json
-proxy {
-  port: 9090
-  port: ${?PROXY_PORT}
-  buffer_size_in_bytes: 524288
-  buffer_size_in_bytes: ${?BUFFER_SIZE_IN_BYTES}
-}
-
-app {
-  uuid: ${?APP_UUID}
-  private_key: ${?APP_PRIVATE_KEY}
-}
+# Mauth Proxy:
+The reason for using the mAuth proxy is to enable the use of UI REST clients like 
+[Postman](https://www.getpostman.com/postman), [Paw](https://paw.cloud/) or curl to request Medidata resources.
+
+The service appends valid mAuth headers to incoming requests and 
+sends the authenticated request to the requested service.
+
+## Running via sbt:
+```bash
+export APP_MAUTH_UUID="<the mauth id of the app you are proxying to>"
+export APP_MAUTH_PRIVATE_KEY="<the actual key (not the path) of the app you are proxying to>"
+./runMauthProxyServer.bash
 ```
 
-## Start the proxy with Docker
+## Running via Docker image
+
+Internal Medidata users please follow [internal guide](https://mdsol.jiveon.com/community/research-development/app-dev/platform-development/services/mauth/blog/2018/03/06/use-mauthed-service-with-a-mauth-unaware-client)
+
+### Installing Docker
+You need to have Docker installed on your machine in order to use Mauth Proxy. The simplest way to get Docker running 
+on a Mac or Windows machine is to follow the instructions at 
+[Docker for Mac](https://docs.docker.com/#/docker-for-mac).
 
-The reason for doing this is so you can use UI REST clients such as [Postman](https://www.getpostman.com/postman) or [Paw](https://paw.cloud/) or even just plain old `curl` to hit Medidata resources
+You can create a default VM for docker and run it by running the following
+commands:
 
-1. Install docker
-  1. `brew install docker`
-1. Start docker host(`docker-machine start <host_name>` or this may already be running if you are using "Docker for Mac/Windows")  
-1. Check out this repository  
-1. cd to the root of this repository
-1. `export APP_UUID='<the mauth id of the app you are proxying to>`
-1. `export APP_PRIVATE_KEY='<the actual key (not the path) of the app you are proxying to>`
-1. `docker build --rm -t mauth-proxy .`
-1. `docker run --env APP_UUID --env APP_PRIVATE_KEY -it --rm -p 9090:9090 --name <your_name> mauth-proxy`
+```bash
+docker-machine create --driver virtualbox default
```

Comment:
With Docker for {Mac|Windows} you don't need to use docker-machine at all. Unless you specifically need to use docker-machine for some reason (it is slightly different in terms of networking), the instructions to create a VM (including this script) can probably be removed. Once you've installed Docker for Mac or for Windows, you get a VM running automatically and I believe the env vars that `eval $(docker-machine env) sets up for you get automatically set up by the Docker installer. It might just require opening a new terminal.